### PR TITLE
Feat: added card component

### PR DESCRIPTION
**Pull request by sushil-mod**
Added cards 
default
card-d
card with text over and card horizontal
card-d-t card-hori
card badge ,shadow and text only
card-badge
card-t-s

### CODE REVIEW

File: Components/card.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    padding: 8px;
+    border-radius: 1rem;
+    border: 1px solid var(--card-border-color);
+    max-width: 250px;
+}
+.card-tag{
+    padding: 4px 8px;
+    border: 1px solid var(--card-tag-border-color );
+    border-radius: 50px;
+    font-size: 12px;
```
Comment (ozairkhan2810) [inline Components/card.css]: use rem

**Review by ozairkhan2810 — APPROVED** [review]
maintain consistency in units, use any one throughout and for below 10px can use in px

File: Components/card.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+} 
+.img-height{
+    height: auto;
+}
+ /* card with badge */
+.card-pos-rel{
+    position: relative;  
+}
+.card-badge{
+    box-sizing: border-box;
```
Comment (tusharkandpal) [inline Components/card.css:65]: you can set `box-sizing` in universal selector `*`

**Review by tusharkandpal — APPROVED** [review]
please go through comment, rest is ok! 👍🏻

**Review by shaksham08 — APPROVED** [review]
Looks good

File: Components/card.css (hunk 1 above)
Comment (sushil-mod) [reply Components/card.css]: done, thanks

File: Components/card.css (hunk 2 above)
Comment (sushil-mod) [reply Components/card.css:65]: done

**Pull request merged**